web/src/api/loki.ts

```diff
@@ -81,6 +83,14 @@ export interface TopologyMetricPeer {
   getDisplayName: (inclNamespace: boolean, disambiguate: boolean) => string | undefined;
 }
 
+export type DroppedTopologyMetrics = {
+  value: number;
```

Comment:
looking only at this type definition, it is unclear what is `value` .. also I can't see any reference to that field, is it unused?

Reply:
correct that's not used anymore
https://github.com/netobserv/network-observability-console-plugin/pull/324/commits/4d1d412011488b8cd012b7ee61fe5be676ccb32f